Dockerfile.local

```diff
@@ -1,32 +1,12 @@
-FROM ubuntu:xenial
+FROM node:6
```

Comment:
@Isabello Shall we stick to the node version used by core?

Reply:
core supports node 6.x.x. However, we should directly use Node 6.11.3.